openedx/core/djangoapps/profile_images/tests/test_images.py

```diff
@@ -0,0 +1,181 @@
+"""
+Test cases for image processing functions in the profile image package.
+"""
+
+from itertools import product
+import os
+from tempfile import NamedTemporaryFile
+
+from django.core.files.uploadedfile import UploadedFile
+from django.test import TestCase
+from django.test.utils import override_settings
+import ddt
+import mock
+from PIL import Image
+
+from ..images import (
+    validate_uploaded_image,
+    generate_profile_images,
+    remove_profile_images,
+    DevMsg,
+    ImageValidationError,
+)
+from .helpers import make_image_file, make_uploaded_file
+
+
+@ddt.ddt
+class TestValidateUploadedImage(TestCase):
+    """
+    Test validate_uploaded_image
+    """
+
+    @ddt.data(
+        (99, DevMsg.FILE_TOO_SMALL),
+        (100, ),
+        (1024, ),
+        (1025, DevMsg.FILE_TOO_LARGE),
+    )
+    @ddt.unpack
+    @override_settings(PROFILE_IMAGE_MIN_BYTES=100, PROFILE_IMAGE_MAX_BYTES=1024)
```

Comment:
i can copy settings to test but i'd still leave this here, because it's very coupled to the assertions being made in the test, but leaving them set like this for all tests would probably cause problems.